doc/sphinx/source/hsa.rst

```diff
@@ -0,0 +1,104 @@
+===
+HSA
+===
+
+Note: pocl's HSA support is currently in experimental stage.
+
+The experimental HSA driver works only with an AMD Kaveri or Carrizo APUs
+using the HSAIL-supported LLVM and Clang. Other than that, you will need
+a recent linux (4.0+) and some software.
+
+Installing prerequisite software
+---------------------------------
+
+1) Install the HSA AMD runtime library
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  Pre-built binaries can be found here:
+
+  https://github.com/HSAFoundation/HSA-Runtime-AMD
+
+  This usually installs into /opt/hsa. Make sure to read Q&A in README.md, it
+  lists some common issues (like /dev/kfd permissions) and run sample/vector_copy
+  to verify you have a working runtime.
+
+2) Build HSAIL-Tools
+~~~~~~~~~~~~~~~~~~~~~
+
+   `git clone` https://github.com/HSAFoundation/HSAIL-Tools
+
+   In particular, **libHSAIL.a** will be required by LLVM, and **HSAILasm** will be
```

Comment:
libHSAIL.a should not be used anymore as we don't want the BRIG output. This breaks pocl build now because libHSAILCodegen.a has it undefined and it's not brought in by pocl's linkage.